dist/donut-chart/donut-chart.css

```diff
@@ -0,0 +1,184 @@
+.donut-chart {
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  max-width: 305px;
+  position: relative;
+  width: 100%;
+}
+.donut-chart__container {
+  display: inline-block;
+  max-width: 120px;
+  width: 50%;
+}
+.donut-chart__stacked .donut-chart__container {
+  max-width: 190px;
+  width: 100%;
+}
+.donut-chart__container-vertical {
+  margin-bottom: 20px;
+  max-width: 200px;
+  width: 100%;
+}
+.donut-chart-legend__container {
+  display: inline-block;
+  width: 100%;
+}
+.donut-chart__stacked {
+  max-width: 190px;
+}
+.donut-chart__legend-left {
+  flex-flow: row-reverse nowrap;
+  justify-content: flex-start;
+  max-width: 450px;
+}
+.donut-chart__legend-left .donut-chart__container {
+  margin-left: 30px;
+  max-width: 190px;
+}
+.donut-chart__legend-right {
+  flex-flow: row nowrap;
+  justify-content: flex-start;
+  max-width: 450px;
+}
+.donut-chart__legend-right .donut-chart__container {
+  margin-right: 30px;
+  max-width: 190px;
+}
+.donut-chart__legend-right .donut-chart-legend__container {
+  max-width: 210px;
+  width: 50%;
+}
+.donut-chart__legend-left .donut-chart-legend__container {
+  max-width: 210px;
+  width: 50%;
+}
+.donut-chart svg {
+  display: inline-block;
+  position: relative;
+  vertical-align: middle;
+  width: 100%;
+}
+.donut-chart__metrics {
+  display: inline-block;
+  width: 50%;
+}
+.donut-chart__legend-left .donut-chart__metrics {
+  width: 100%;
+}
+.donut-chart__legend-right .donut-chart__metrics {
+  width: 100%;
+}
+.donut-chart__stacked .donut-chart__metrics {
+  display: block;
+  margin-bottom: 30px;
+  width: 100%;
+}
+.donut-chart__horizontal-metrics {
+  margin-bottom: 20px;
+}
+.donut-chart__metric {
+  display: block;
+  font-size: 30px;
+  font-weight: 700;
+  line-height: 40px;
+}
+.donut-chart__subtitle {
+  display: block;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 18px;
+}
+.donut-chart__segment {
+  opacity: 1;
+  transition: opacity 0.3s;
+}
+.donut-chart__segment--primary {
+  fill: var(--data-viz-primary, var(--color-data-viz-chart-primary));
+  stroke: var(--data-viz-primary, var(--color-data-viz-chart-primary));
```

Comment:
So generally the way we handle variables is this:

The first one is going to be a token variable, which is per component. This allows a user to override colors for this component only. The second one is fine, it's the one used in the tokens file.
The first variable should not be used in the code at all. This is part of a future project

So I would change this as:
`fill: var(--donut-chart-color-primary, var(--etc))`
Or something like that. See https://github.com/eBay/skin/blob/master/src/less/button/button.less#L98


I would also add, lets use a mixin for this, or add one if there isn't (I see fill has one, but stroke does not)
https://github.com/eBay/skin/blob/master/src/less/mixins/private/token-mixins.less#L81